packages/builders/package.json

```diff
@@ -77,7 +77,7 @@
 		"prettier": "^2.5.1",
 		"tsup": "^5.11.13",
 		"typedoc": "^0.22.11",
-		"typescript": "^4.5.5"
+		"typescript": "^4.6.1-rc"
```

Comment:
do we really wanna use a TS rc? Also what for

Reply:
This was not supposed to be in the PR, I was testing something.